translations/Arabic/special-terms.md

```diff
@@ -0,0 +1,48 @@
+# Terms
+
+- Message: رسالة
+  لقد وصلتك رسالة
+- Private Message (PM): رسالة خاصة
+  أرسل رسالة خاصة
+- Group PM: رسالة خاصة لمجموعة
+- Realm: عالم
+  Out of context, it would mean 'kingdom', so the word for 'world' is better here
+- Stream: بث
+  Direct translation would be something like river stream. بث is the best translation in the context
```

Comment:
It means broadcast (as in TV broadcast), but it's the word always used in Arabic Apps for "[Live] streams"